Samples/Windowing/cpp-winui/SampleApp/DemoPage.xaml.cpp

```diff
@@ -47,5 +47,20 @@ namespace winrt::SampleApp::implementation
             m_mainAppWindow.Resize(windowSize);
         }
     }
-  
+    void DemoPage::ClientSizeBtn_Click(winrt::Windows::Foundation::IInspectable const& sender, winrt::Microsoft::UI::Xaml::RoutedEventArgs const& e)
+    {
+        int windowWidth = 0;
+        int windowHeight = 0;
+
+        windowWidth = _wtoi(WidthTextBox().Text().c_str());
+        windowHeight = _wtoi(HeightTextBox().Text().c_str());
```

Comment:
It is ok to declare and assign value to a variable at the same time, please consider simplifying:
        int windowWidth = _wtoi(WidthTextBox().Text().c_str());
        int windowHeight = _wtoi(HeightTextBox().Text().c_str());
OR consider defining the windowSize variable here and use its Width/Height properties directly instead of using intermediary variables:
        winrt::Windows::Graphics::SizeInt32 windowSize;
        windowSize.Width = _wtoi(WidthTextBox().Text().c_str());
        windowSize.Height = _wtoi(HeightTextBox().Text().c_str());
and updating the if-statement below accordingly.